### DISCUSSION TITLE
Grades

### DISCUSSION THREAD

**Discussion by emign** [RafaelGB/obsidian-db-folder#430] [Q&A]
Hi,
is there any way to use a db as a grading system for education ?
For example:
- You predefine grades (1-10, A-F, * - *****)
- any column with type grade has cells which offer a drop-down (like current selection cells) 
- when you choose a grade it appends the grade into metadata together with the current time stamp

Kind regards

**Comment by RafaelGB** [comment]
The possibility of create tools like that have the base with the formulas.

Currently you can detects changes per row and run a formula with the context of the row using javascript (There is a section into the documentation)

With the time we will improve those formulas in function of the feedback, sure 😄
